tools/gcpviz/wait_for_export.sh

```diff
@@ -0,0 +1,24 @@
+ #!/bin/bash
+
+ wait_for_export() {
+          >&2 echo "Running: $@"
```

Comment:
What's the expected output of $@ in this context ?

Reply:
```$ ./wait_for_export.sh gcloud asset export --output-path=gs://$CAI_BUCKET_NAME/resource_inventory.json --content-type=resource --organization=$ORGANIZATION_ID --billing-project=pgsz-sharedsvcs-hub
Running: gcloud asset export --output-path=gs://cust1-access-test-bucket/resource_inventory.json --content-type=resource --organization=1234567890 --billing-project=foobar
Output: Export in progress for root asset [organizations/312281222529].
Use [gcloud asset operations describe organizations/1234567890/operations/ExportAssets/RESOURCE/d9359e9bd89944ae2b194267d6ce26de] to check the status of the operation.
Status command: gcloud asset operations describe organizations/1234567890/operations/ExportAssets/RESOURCE/d9359e9bd89944ae2b194267d6ce26de
```

Reply:
Changed it to $* due to linting errors.